test/team_stats_test.rb

```diff
@@ -111,6 +111,18 @@ def setup
 #     }
 #     assert_equal expected, @stat_tracker.head_to_head("18")
 #   end
+<<<<<<< HEAD
```

Comment:
Is there a merge conflict here?

Reply:
It looks like there was but the data is the exact same so IDK but I will clean it up to make it look presentable